stock_delivery_result_import/wizards/stock_delivery_result_import.py

```diff
@@ -32,6 +33,7 @@ def import_stock_delivery_result(self):
                 picking = picking_obj.search(
                     [("name", "=", picking_ref), ("company_id", "=", company.id)]
                 )
+                carrier_tracking_ref = row_dict.get("carrier_tracking_ref")
```

Comment:
@TomohiroAbe-Quartile This is not enough.  We need to consider the case where a picking contains multiple tracking refs (multiple lines with the same picking ref).

Reply:
I tried like below:
Import CSV contains like: 2 deliveries, each with 3 delivery slip numbers.

→Confirmed 2 deliveries contain each with 3 slip numbers.